index.md

```diff
@@ -65,10 +65,20 @@ In the meantime, publication of Internet-Draft documents can be tracked through
 * [Relative JSON Pointers](https://datatracker.ietf.org/doc/draft-bhutton-relative-json-pointer/)
 
 Internet-Drafts expire after six months, so our goal is to publish often enough to always have a set of unexpired drafts available.  There may be brief gaps as we wrap up each draft and finalize the text.
+</details>
 
-The intention, particularly for vocabularies such as validation which have been widely implemented, is to remain as compatible as possible from draft to draft.  However, these are still drafts, and given a clear enough need validated with the user community, major changes can occur.
+### Use of the _draft_ designation
 
-</details>
+Releases of the JSON schema specification and meta schemas use the _draft_ designation primarily for historical reasons stemming from the relationship of this specification to IETF ([explained here](https://json-schema.org/specification-links.html#understanding-draft-names-and-numbers)).
+The use of this designation is under review but will continue until this review process completes to avoid changing the designation style multiple times.
+
+The JSON schema project recognizes, condones and advocates for the use of the JSON schema standard in production.
```

Comment:
```suggestion
The JSON schema project recognizes, condones, and advocates for the use of the JSON schema standard in production.
```

Reply:
Nit pick. Oxford comma.

Otherwise, This is good to go!
Thanks @aucampia for taking the time to make a PR here! 👌

Reply:
Change applied